test/future.js

```diff
@@ -0,0 +1,133 @@
+'use strict';
+
+const { Future, futurify } = require('..');
+const metatests = require('metatests');
+
+metatests.test('Future map/run', async test => {
```

Comment:
```suggestion
metatests.test('Future map/run', test => {
```
?

(and in other tests)